feature-libs/cart/saved-cart/components/list/saved-cart-list.component.html

```diff
@@ -2,11 +2,17 @@
   <ng-container *ngIf="!(isLoading$ | async); else loading">
     <div role="status" [attr.aria-label]="'common.loaded' | cxTranslate"></div>
     <div class="cx-saved-cart-list-header">
-      <h3>
+      <!-- TODO:  for next major releasee remove below h3 tag and feature level check from the h2 tag-->
```

Comment:
I would encourage you to create a ticket for the TODO change, referencing the ticket number here in the comment.   Please set the expected release version in the ticket.

See https://github.com/SAP/spartacus/blob/develop/feature-libs/asm/components/customer-selection/customer-selection.component.spec.ts#L142 for an example.